Revised manual uploading steps

joshg2k opened the pull request:
Revised the *iOS Manually uploading your app* section since Application Loader is no longer available in XCode 11. Consolidated these same steps from the *Start the upload* section to remove redundancy.

# Why
Application Loader was removed from XCode 11, so these steps were no longer accurate. I replaced them with the equivalent command line arguments until a better solution is found.

# How
I identified the command line equivalents that Application Loader was running, and used them to simulate the previous Application Loader workflow. I have tested these steps a dozen or so times with my own iOS app submissions.

# Test Plan
Follow the steps detailed in this change set and verify that IPA builds submits to iOS App Store properly.

joshg2k committed:
Revised manual uploading steps

Revised the *iOS Manually uploading your app* section since Application Loader is no longer available in XCode 11. Consolidated these same steps from the *Start the upload* section to remove redundancy.

brentvatne commented:
looks good, thank you! can you also copy these changes to the `unversioned` version of this doc?

joshg2k committed:
Updated unversioned documents for future revisions

I accidentally updated a versioned form of the docs without also keeping the latest unversioned copy updated. This commit brings the unversioned HEAD up to date.

joshg2k commented:
@brentvatne done. Thanks for pointing that out, I'll make sure my future edits are done in all the appropriate places.

natelowry reviewed (approved):
works4me! thanks for adding this!

brentvatne commented:
thank you 🙏
